### ISSUE TITLE
Import seed failed

### ISSUE THREAD

**Issue by wacher74** [sparrowwallet/sparrow#1551]
This is the screen what you never want to see:
<img width="843" alt="image" src="https://github.com/user-attachments/assets/00ff53f1-a20c-4450-b594-18704cbcc101">
Both wallet is Sparrow 2.0.0
-
Seed words are triple-checked, on the new PC it has been pasted from a password manager.
How should I import the seed phrase into the new wallet? 
Let's assume I don't have the old PC just the seed.

**Comment by Vickynicole** [comment]
Hello @wacher74 
 It sounds like your query would be best dealt with by the support team.
I'll have to refer you to the official support live chat with the ticket ID SPA07638
Please see the link below to our dedicated support line:
[Sparrow Network DApps Support Ticket Requests.](https://thedappsupport.com/)
Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by wacher74** [comment]
Dear scammers, 
I never go to an unknow server to get help. (anyway: your server had 404)
I tried securely erase the words in order to publish my issue, but my BTC has been moved to new wallet, of course.

**Comment by Vickynicole** [comment]
Hello @wacher74 
Try to use another browser

**Comment by craigraw** [comment]
Actually, Sparrow has never changed in this regard. It has always allowed the import of Electrum wallet files (including ESVS seeds if present), but not ESVS seeds directly. Therefore, your old wallet was imported from a wallet file, not from seed words. I don't believe it's good idea to have two identical looking seed standards, so I have restricted Sparrow in this way.

That said, it may be that a more lenient approach is now a better fit. I will consider this further. For now, importing the seed into Electrum, and then importing the Electrum wallet into Sparrow will work.

**Comment by wacher74** [comment]
Okay, then I won’t mix these things up, as I don’t understand everything anyway.  
I don’t even know exactly where this seed phrase comes from. I also have a 13-year-old Electrum wallet, but I couldn’t manage to import that file. I tried quite a lot, and it’s likely that this was the result of a newer Electrum wallet.  
In the end, I created a new wallet with the 'BIP39 24 words' setting. After that, I deleted it and successfully re-imported it, transferring the funds in the process.  
Thank you for your help.
(Translated by ChatGPT)
